OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs

```diff
@@ -58,6 +58,9 @@ public int Team
 		[Desc("Returns true if the player is the local player.")]
 		public bool IsLocalPlayer { get { return Player == (Player.World.RenderPlayer ?? Player.World.LocalPlayer); } }
 
+		[Desc("Returns the player actor of this player.")]
+		public Actor PlayerActor { get { return Player.PlayerActor; } }
```

Comment:
Please don't leak the player actor to scripts - we can define a `GrantPlayerCondition` via a new `ScriptPlayerProperties` subclass instead?

Reply:
I don't exactly see a problem right now with this and consider it useful since there are potentially more things than granting conditions one would want the player actor for, but I guess those can be added to `ScriptPlayerProperties` instead if needed.